fvm/evm/precompiles/signature.go

```diff
@@ -0,0 +1,35 @@
+package precompiles
+
+import (
+	"fmt"
+	"strings"
+
+	gethCrypto "github.com/ethereum/go-ethereum/crypto"
+)
+
+const FunctionSignatureLength = 4
+
+// This is derived as the first 4 bytes of the Keccak hash of the ASCII form of the signature of the method
+type FunctionSignature [FunctionSignatureLength]byte
+
+func (fs FunctionSignature) Bytes() []byte {
+	return fs[:]
+}
+
+// ComputeFunctionSignature computes the function signture
+// given the canonical name of function and args.
+// for example the canonical format for int is int256
+func ComputeFunctionSignature(name string, args []string) FunctionSignature {
```

Comment:
do we need these two (`SplitFunctionSignature`, `ComputeFunctionSignature`)  exported? it looks like we don't at this point

Reply:
I exposed them for testing and I think there might be other places that we are going to use them, given they are very well known in the EVM community. I fixed the name to use Selector, Signature was something else :D